### PR TITLE
Removes Cult and Wizard from the game.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

See title.

## Why It's Good For The Game

Cult and Wizard are really, really bad game modes that we've left in for way too long, and it's time for them to go.

After over 5 reworks, Cult still isn't a fun experience and still suffers from the same core problems of it being a conversion TDM game mode with a forced round end condition. We've tried so many different cult variants, so so many, and it just isn't fun every time because the same core flaws aren't fixed. The gameplay loops end up being the same every time.
The cult either:
1. Gets caught early, dies, everyone gets implants and maintenance gets purged of cultists.
2. Doesn't get caught early, converts half the station, round ends.

Occasionally they get to do some magic power stuff except like, 90% of them aren't good or interesting, and the 10% that are good are just reviving dead player magic. Then the round ends or the cult dies. It's not fun for the majority of the people involved.

Wizard is a really bad game mode for a multitude of reasons.
1. It's a solo antagonist, so the entire driving force of the shift is placed on this one guy, if no other rulesets have been rolled.
2. It's destructive enough as a solo antagonist that it overshadows every other antagonist in terms of intensity, importance, and deadliness, making it play not so nice with Dynamic.
3. In the hands of an experienced player, a Wizard can kill the entire fucking station within 5 minutes. It's not fun for anyone *but* the wizard.
4. Since it's a solo antagonist, the player in charge of the solo antagonist can and often does opt to not actually do their job as an antagonist, and can just sit around the map doing nothing all day if they want to, and the crew won't call the shuttle as a result.

The last point may not seem like an issue, but look at every other antagonist.

- Traitors, Changlings, and Heretics have objectives to escape alongside their objectives to cause chaos, so they'll try to get the shuttle called, usually via some dangerous shit or espionage if left alone.
- Nuke Ops and Blobs are destroying the station with a bomb, so they can't be ignored and therefore the crew has to move the round forward by taking care of the antag, or they all die.
- Revolutionaries are actively converting the entire crew to be revolutionaries, and then killing the heads of staff.

Every other antagonist has some reason to be dealt with that results in the crew having to deal with them, even if they're sitting around doing nothing. Wizards don't, and can just sit around endlessly with no problem.

On top of that, Friendly wizards pose a problem for any other antagonists picked by Dynamic because no antagonist in the game is prepared to deal with a wizard. They just do not match up, at all, without some serious combat cheese or an extreme ambush that prevents a fight in the first place. A wizard can solo all the revolutionaries. A wizard can solo a nuke ops squad. The only antagonist immune to the wizard is Blob, and even then that's a toss-up since the wizard can fireball.

In summary, removing Wizard and Cult will improve the game as people will not have to be involved with those round types anymore.

Yes, I am aware some of this criticism applies to Families as well, so let's not litigate that in this PR. **Please keep discussion to Cult and Wizard. Posts about other gamemodes will be reported as off topic to project staff.**

## Changelog
:cl:
del: Cult and Wizard are no longer rulesets.
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
god damn it goof now the reddit post will have something to point to, and they'll go off on tangents again

**Comment by Mothblocks** [comment]
image

**Comment by Iamgoofball** [comment]
please do not off topic shitpost on my PR, i can't hit the report offtopic button to send it to oranges anymore

**Comment by TheMidnightRose** [comment]
So question. What replaces them then?

**Comment by Colovorat** [comment]
> So question. What replaces them then?

Clockwork cult

**Comment by drIcepop** [comment]
> > So question. What replaces them then?
> 
> 
> 
> Clockwork cult

That's a good one

**Comment by SamDDBS** [comment]
april fools was last month

**Comment by 81Denton** [comment]
Tocuhes config, DNM until headmin approval

**Comment by badlypayedgoon** [comment]
So you don't like them, and that is why you want to delete them for other players that might like them? And you do not even attempt to add 2 game modes in return to fill the void? Yeah, no.

**Comment by Colovorat** [comment]
> So you don't like them, and that is why you want to delete them for other players that might like them? And you do not even attempt to add 2 game modes in return to fill the void? Yeah, no.

Show me at least one who likes them

**Comment by Nexofas** [comment]
Any ideas with die of fate 20 roll?

**Comment by Fikou** [comment]
this is cringe if youre not gonna go and actually put effort in removing deprecated cult and wizard code

**Comment by Iamgoofball** [comment]
@Fikou left it in for admin abuse

**Comment by Colovorat** [comment]
unkn1own

**Comment by MaxIsJoe** [comment]
Can't fix the problem if it doesn't exist anymore.

**Comment by jlsnow301** [comment]
> Gets caught early, dies, everyone gets implants and maintenance gets purged of cultists.

How is revs any different than this? Conversion may be a bad design but at least cult gets more than a flash

**Comment by Colovorat** [comment]
> > Gets caught early, dies, everyone gets implants and maintenance gets purged of cultists.
> 
> How is revs any different than this? Conversion may be a bad design but at least cult gets more than a flash

No?

**Comment by Colovorat** [comment]
image
help me i'm being hit with bad slimes, now my opinion is bad and i'm going to reddit to rant about it

**Comment by MaxIsJoe** [comment]
> help me i'm being hit with bad emojis, now my opinion is bad and i'm going to reddit to rant about it

I don't know what kind of gotcha is this but if you're getting "negative" emojis along-side other negative comments about this that means there are people out there who don't like/agree your comment and your PR (and your attitude). 

But you know what? I agree. People should flood this PR more with negative comments instead of meanie emojis that does not provide any sort of criticism just so the point can go across that this is a terrible change and the majority does not like it at all.

**Comment by EgorDinamit** [comment]
Please ignore this PR, it's obviously his plan to create yet another drama.
I do realize that they may actually merge it, but for now it's just a goof PR.

You can just look at this.
image

And out of almost a thousand of PRs, only 295 are merged.
image


Conclusion? It's just a drama PR, similar to #58547, #31576 and #28212

**Comment by SmArtKar** [comment]
Comment em out

**Comment by LemonInTheDark** [comment]
> 
> 
> Comment em out

If we ever do this I'm leaving

**Comment by CRiPLe1000** [comment]
Its not worth it but fuck it, lets attack it one by one:

> The cult either:
Gets caught early, dies, everyone gets implants and maintenance gets purged of cultists.
Doesn't get caught early, converts half the station, round ends.

Same can be said about Revolutionaries in even worse version as success of Revs depends on Head Revolutionaries being able to convert and not get caught early. If you have 3 Headrevs that got caught early because of bad luck, bad play or not knowing how to play it, you have the same stuff. It also works the other way around, when Headrevs stealth convert half the station and then steamroll sec and Heads.
If those are the reasons, why not remove Revs too? And yeah, despite what you have told about not including Families, i will do it anyway cause they had the same problems with gameplay loop when they were still on LRP - get antag and kill everyone else until Space Cops arrive and either steamroll you or you kill them. Same stuff every round.

> Occasionally they get to do some magic power stuff except like, 90% of them aren't good or interesting, and the 10% that are good are just reviving dead player magic. Then the round ends or the cult dies. It's not fun for the majority of the people involved.

While I don't disagree with the statement that there are some useless spells (Hallucinations, Conceal Presence for instance), almost every spell outside of them is used during cult rounds. Stun, Teleport, EMP are main toys for cult, while twisted construction or combat equipment are used at least once a round by experienced cultists. Same can be said about Runes, with exception of Blood Boil (i haven't seen it used, so it might be me) or Apocalypse, which not only is useless as it can actively fuck up Cult (le funny Rod/Rad Storm) or doesn't help at all (Disease Outbreak + Mice Migration/Anomalies/Grey Tide).
And here is a kicker - yet again Revs don't get jack shit, here is a bat that i crafted from sawdust and apply it to some security officer. This alone makes Cult more interesting to play with/against it then Revs and you want to delete it, just stop.

Im starting to regret writing response and Im only halfway there...

> Wizard is a really bad game mode for a multitude of reasons.
> It's a solo antagonist, so the entire driving force of the shift is placed on this one guy, if no other rulesets have been rolled.

Yeah, i agree with it, however i would rather pin the problem on cost of the Wizard then it being pinned on one guy. Antags are enemies to themselves unless they agree for ceasefire and roll together, which with Wizard killing everyone in sight is rare. I would like to see how Wizard is played with larger amount of threat, how other antagonists interact with each other.

> It's destructive enough as a solo antagonist that it overshadows every other antagonist in terms of intensity, importance, and deadliness, making it play not so nice with Dynamic.

Thats why it costs a lot of threat to be spawned and yet again, try to kill fire(ball) with (gun)fire and see how it works. Also we should go with the threat price, not how many people takes a role and, if I recall correctly, Wizard is on par with Revs, Nukies and Cult, so make up your mind and stop cherry picking Antags that you don't like.

> In the hands of an experienced player, a Wizard can kill the entire fucking station within 5 minutes. It's not fun for anyone but the wizard.

In hands of experienced traitor it will take 10 minutes if we take the same number of pop as with 5 minutes example which is... 20 people? Same can be said about every other antag anyway (that can actually spawn on low pop). Also, even experienced Wizard can be met with experienced Security (or anyone tbh) and die within few minutes (batong, bat, flash, welderbombing, shoving and stealing their weapon if they get Mjolnir, etc. there are endless ways to do it). Wizard is a Glass Cannon that can kill a lot of people but can die easily. However, people (or rather me, but I wouldn't be surprised if other would too) hate Wizards that cheese it by taking Lich and dropping their bounded item in Space outside Station with little to no indication where it really is (add GPS signal that activates for brief moment after Lich dies to stop the bullshit).
Fighting the Wizard can be fun, the never ending chase (as its mostly the Wizard that runs away) has its interesting moments. And even then, if Wizard is too powerful, Admins can send ERT, giving second opportunity for players (tho they might fuck up Wizard too hard, sometimes it happens).

> Since it's a solo antagonist, the player in charge of the solo antagonist can and often does opt to not actually do their job as an antagonist, and can just sit around the map doing nothing all day if they want to, and the crew won't call the shuttle as a result.

This again... all antagonists can decide to do that (outside of team antags like Cult or Revs, Nukies can do stealth and die outside of the shuttle by forgetting to switch to EVA mode) and while it may make round more boring, pricing of the threat may be the solution, dropping the price of Wizard so other antags may coexist (before killing each other). Meanwhile changing the rules, forcing people to do objectives as solo antags will definitely be met with positive response (its a joke). Anyway, if Wizard decides to explore Space and fuck off Im pretty sure that its a ground for Admin intervention to at least offer body of the wizard to someone else and respawn player with some basic stuff in the space so they can do what they want. Its the same way as the only Station Engineer not setting up SM/Solars but taking hardsuit and going into space.

> The last point may not seem like an issue, but look at every other antagonist.
[SNIP]

Look above

>  A wizard can solo all the revolutionaries. A wizard can solo a nuke ops squad. The only antagonist immune to the wizard is Blob, and even then that's a toss-up since the wizard can fireball.

Revs are the same as crew but more prepared to fight by definition as they get blood thirsty as soon as they get flashed.
But nuke ops? Excuse me? I mean sure, Wizard can solo nuke ops but how many times? What are the chances? Theoretically I can solo entire USA Army by standing and throwing sticks at them, but realistically the chance is really low. Wizard can die easily as they don't have "burst" healing, with little to no protection against damage (with Nukies dealing a lot of burst damage), so I wouldn't be surprised if Wizard would see Nukie, try to Fireball it and then get shot to death by the same Nukie. I would really like to see how the fight would roll out to see how strong are Antags against each other. The more you know...

> In summary, removing Wizard and Cult will improve the game as people will not have to be involved with those round types anymore.

They will, as other Antagonists that can do basically the same still exists.

> Yes, I am aware some of this criticism applies to Families as well, so let's not litigate that in this PR. Please keep discussion to Cult and Wizard. **Posts about other gamemodes will be reported as off topic to project staff.**

No. If there is a problem in design wise, find all instances of that problem and then solve it. 

PS:
I was told that you made some time ago (1 year at least) design doc for a "replacement" of a cult and its really interesting to read last two sentences of it with combination of basically the first sentence of this PR.

> Their objective is to kill every last Fanatic on the station, and then cover up the Fanatic remains by detonating the station nuke.
Upon killing the last Fanatic, they will be given the nuclear code, and tasked with getting dat fukken disk.

> After over 5 reworks, Cult still isn't a fun experience and still suffers from the same core problems of it being a conversion TDM game mode with a forced round end condition.

Link to hackmd: https://hackmd.io/@tgstation/HJG54zYoL

**Comment by Iamgoofball** [comment]
>No. If there is a problem in design wise, find all instances of that problem and then solve it.

I'm not saying there isn't design problems with other gamemodes, I'm just trying to keep this PR to Cult and Wizard specifically. The rest will get PRs eventually.

**Comment by DrCrawler** [comment]
Baisicly just a rant PR. If you dont like the amount of action packed in a single round go to manuel or some HRP server. If you REALY want to get rid of the 2 antags go get a replacement. Oh and if you even _try_ to copypaste some random antag from another server or get the clockwork cult back people are gonna quickly shut the thing down because of the thing i like to call "boredom". /tg/ community wants ACTION and lone antag based rounds. If something is gonna change in the near future its probably minor balance and tweaks to antags (Honestly tho new tools for tator and new spells for wizard would be absolutely amazing). Oh and there is like this russian server with cult based on RP more than action, dunno if the crowd will like it but for me it suits manuel pretty good.

> > So you don't like them, and that is why you want to delete them for other players that might like them? And you do not even attempt to add 2 game modes in return to fill the void? Yeah, no.
> 
> Show me at least one who likes them

Litturaly me.

**Comment by DrCrawler** [comment]
> > If you dont like the amount of action packed in a single round
> 
> Did you even read the OP?

languege problems dunno how to explain my point here properly.

**Comment by Iamgoofball** [comment]
The proposal for removing cult and wizard has nothing to do with "action", it's about the core design flaws with the gamemodes.

**Comment by DrCrawler** [comment]
Name 3 flaws that only cult and wizard gamemodes experience.

**Comment by Iamgoofball** [comment]
>Name 3 flaws that only cult and wizard gamemodes experience.

Once again, did you read the OP?

**Comment by CRiPLe1000** [comment]
> Name 3 flaws that only cult and wizard gamemodes experience.

He cant as other antags also have them, sometimes in even worse versions.

**Comment by Colovorat** [comment]
> Name 3 flaws that only cult and wizard gamemodes experience.

John you fucking know this flaws, it's unfunny fights, when one side is too superiour to other. it's cool to play as cult, but have you ever tried to fight cult? Getting insta stunned and then boring shit goes? Have you ever tried to fight wizard and not sit in cargo, when you loose all your limbs to one fucking fireball? Even if you have some shit for wizard he has so many escapes, same as cultists, teleport spells, they can be revived, and other shit

**Comment by DrCrawler** [comment]
> It's a solo antagonist, so the entire driving force of the shift is placed on this one guy, if no other rulesets have been rolled.

- Traitor
- Heretic
- Nuke ops (same shit but more people)
- Lone Op (midround, still 1 guy. Can end the round in addition)

> It's destructive enough as a solo antagonist that it overshadows every other antagonist in terms of intensity, importance, and deadliness, making it play not so nice with Dynamic.

- Nuke ops
- Sometimes heretic
- Rarely traitor (If they manage to invoke a full on genocide untill the point when they have top weapons, armor etc.)
- Lone op (again, midround but i dont realy care its same shit but with guns and eswords instead of spells)

> In the hands of an experienced player, a Wizard can kill the entire fucking station within 5 minutes. It's not fun for anyone but the wizard.

- Traitor
- Heretic
- Lone op (blah blah same shit blah blah)
- Nuke Ops
- Fucking anyone if they decide that they dont want to return to tg anymore, but want to go out with a bang

> Since it's a solo antagonist, the player in charge of the solo antagonist can and often does opt to not actually do their job as an antagonist, and can just sit around the map doing nothing all day if they want to, and the crew won't call the shuttle as a result.

Read rule 4.


Also there is a thing like (s)laughter demons and apprenteges, flesh to stone plus staff of animation and occational teaming up with other antags.

Go ahead, remove ALL the antags if you want to. Each and every point you try to prove right ends up to be relevant to many other antagonists. You need to come up with something more creative than just removing something only you dont like. If the playerbase didnt like it there would be like 10 billion more PRs like this.

Im just gonna go ahead and leave this PR alone for good because i already know whats gonna happen to it.

**Comment by MMMiracles** [comment]
this seems like the kind of pr that'd get merged and immediately reverted just so it pops up as merged in ooc and people get mad about it

**Comment by Mothblocks** [comment]
i can just do that with a global announce though

**Comment by Acer202** [comment]
Getting silently hard-stunned and sharded has to be the worst possible SS13 experience. I wouldn't miss the "fun" of a 20-minute shift due to a steamrolling cult for a moment.

**Comment by EgorDinamit** [comment]
Go ahead, remove every gamemode. 
Dying to an experienced wizard is the same as dying to an experienced [insert antagonist name].

**Comment by FloranOtten** [comment]
> Show me at least one who likes them

 I'm a one. I think.

**Comment by Qustinnus** [comment]
> @Fikou left it in for admin abuse

laziest excuse I've heard in a while.

**Comment by necromanceranne** [comment]
Finally, a good goofball pr

**Comment by celotajstg** [comment]
Honestly, I think goof has a point here.

**Comment by Mothblocks** [comment]
then it should be easy to convince the head admins

**Comment by Dorsisdwarf** [comment]
> 
> 
> Heretic is what cult should always have been.
> 
> Make cult group heretic.

One guy going door to door with no sign he's a baddie exterminating the entire station before any of them can call for help in order to get a super duper buff to help him murderbone even harder?

**Comment by wesoda25** [comment]
They're both bad game modes but I find them fun (for the most part) and wizard even a little endearing. 

Wiz is definitely in a better place in dynamic, I don't see how your second point is an issue. Having antags that operate at different power levels is interesting imo. Some spells could go, though. 

I'd say cult is the main culprit for not working well with dynamic, since it still operates on binary win/lose round end conditions.

**Comment by SuperNovaa41** [comment]
This is like, the laziest removal I have ever seen.

**Comment by MrStonedOne** [comment]
> the same core problems of it being a conversion TDM game mode

Why is this bad.

I liked revs, I've played on both sides of revs and liked both ends of it.

I liked old cult (never played on new cult), I've played both sides of it and enjoyed both matches.

If you can't make your argument without buzz words then leave the prs to somebody else to make.

**Comment by gurfan** [comment]
based

**Comment by Nexofas** [comment]
> Any ideas with die of fate 20 roll?

?

**Comment by ATH1909** [comment]
> > So you don't like them, and that is why you want to delete them for other players that might like them? And you do not even attempt to add 2 game modes in return to fill the void? Yeah, no.
> 
> Show me at least one who likes them

me, they're my favorite gamemodes.

**Comment by ATH1909** [comment]
> Since it's a solo antagonist, the player in charge of the solo antagonist can and often does opt to not actually do their job as an antagonist, and can just sit around the map doing nothing all day if they want to, and the crew won't call the shuttle as a result.

Except it's no longer a solo antagonist, since we only have dynamic mode now. A wizard dying or not doing much in dynamic is basically equivalent to the shift having rolled a low starting threat level, which is okay.

**Comment by Sparkezel** [comment]
I love cult, I love heretic, Wizard is fun

**Comment by YakumoChen** [comment]
Every video of @MrMelbert robusting tg as a wizard is like a priceless treasure

Dont take this away from me

**Comment by Pugie** [comment]
This kind of balancing obsession is what ruins tg! Hmmyes, ok right uh huh. Stun removal this, bell curve that, ramblings of insane self proclaimed 'game design experts' Design an OS for a sentry gun IFF and throw all the coders in front of it and see what happens I do say! This balance humdrum is simply deranged and foolish, the only other person to attempt balance on this level was purple had a huge chin and killed spiderman so this is clearly the wrong move. HEATHENS.

**Comment by optimumtact** [comment]
I like em

**Comment by optimumtact** [comment]
> 
> 
> This kind of balancing obsession is what ruins tg! Hmmyes, ok right uh huh. Stun removal this, bell curve that, ramblings of insane self proclaimed 'game design experts' Design an OS for a sentry gun IFF and throw all the coders in front of it and see what happens I do say! This balance humdrum is simply deranged and foolish, the only other person to attempt balance on this level was purple had a huge chin and killed spiderman so this is clearly the wrong move. HEATHENS.

1 million rounds on tg and not a sign of a 50% win rate, they have played us for absolute fools

**Comment by SmithJ0hns0n** [comment]
> Show me at least one who likes them

I like both of em

**Comment by SamDDBS** [comment]
> 
> 
> I like em

based oranges, you have my respect

**Comment by carlopault** [comment]
TG has been ruined sad but true

**Comment by d3athrow** [comment]
dare i say oranges is based?

**Comment by KirbytheWiz101** [comment]
Holy absolute shit, this is why I don't play on TG anymore. Absolute whiney faggots who have no game sense at all and are out of touch with what the playerbase as a whole once. Oranges is a tranny faggot who should join the 50% and goof  ball here has a literal dent in their skull. Keep doing this please, it makes me laugh.

**Comment by wesoda25** [comment]
Sneed

**Comment by interestingusernam3** [comment]
Chuck

**Comment by EgorDinamit** [comment]
I warned you. It was their plan all along.

**Comment by Mothblocks** [comment]
https://www.youtube.com/watch?v=T1XgFsitnQw
